src/Sarif.Viewer.VisualStudio/Converters/CallTreeNodeToTextConverter.cs

```diff
@@ -31,44 +26,25 @@ public object ConvertBack(object value, Type targetType, object parameter, Syste
         private static string MakeDisplayString(CallTreeNode node)
         {
             // Use the following preferences for the CallTreeNode text.
-            // 1. AnnotatedCodeLocation.Message
-            // 2. AnnotatedCodeLocation.Snippet
-            // 3. Callee for calls
-            // 4. "Return" for returns
-            // 5. AnnotatedCodeLocation.Kind
+            // 1. CodeFlowLocation.Location.Message.Text
+            // 2. CodeFlowLocation.Location.PhysicalLocation.Region.Snippet.Text
+            // 3. "<unknown callee>"
```

Comment:
Ah, this is what I was saying about. Just say "Continuing..." or the like. You don't know that there's any calling involved.

Reply:
Done